Source/WebCore/page/SettingsBase.h

```diff
@@ -166,8 +166,10 @@ class SettingsBase : public AbstractRefCounted {
     void shouldUseModernAVContentKeySessionChanged();
 #endif
     void useSystemAppearanceChanged();
+    void fontFallbackPrefersPictographsChanged();
 
     WeakPtr<Page> m_page;
+    RefPtr<Page> protectedPage() const;
```

Comment:
Please do not add member functions in the middle of data members. This is definitely NOT the pattern in WebKit code.